pkg/yurthub/filter/constant.go

```diff
@@ -46,6 +46,11 @@ const (
 	// If end users want to use specified LB service at the edge side,
 	// End users should add annotation["openyurt.io/skip-discard"]="true" for LB service.
 	SkipDiscardServiceAnnotation = "openyurt.io/skip-discard"
+
+	// DiscardServiceAnnotation is annotation used by LB service.
+	// If end users want to discard specified LB service at the edge side,
+	// End users should add annotation["openyurt.io/discard"]="true" for LB service.
+	DiscardServiceAnnotation = "openyurt.io/discard"
```

Comment:
openyurt.io/discard --> svc.openyurt.io/discard

Reply:
That's right, fixed.